mongodbatlas/mongodbatlas.go

```diff
@@ -64,6 +66,7 @@ type RequestDoer interface {
 	Doer
 	Completer
 	NewRequest(context.Context, string, string, interface{}) (*http.Request, error)
+	NewRequestAndSetBaseUrl(context.Context, string, string, string, interface{}) (*http.Request, error)
```

Comment:
The idea is to remove `NewRequest` and use only `NewRequestAndSetBaseUrl` that takes the API version as input

Reply:
oh I think I know what you want but no, let's not go this, this is now the time for something I've wanted a long time ago, let's make `defaultBaseURL="http://cloud.mongodb.com/"` and let each group of APIs include the `api/atlas/v1.0/` on their paths

Reply:
Got it! Thanks! : )